servers/Route.bones

```diff
@@ -24,7 +24,8 @@ servers['Route'].augment({
             require.resolve('../assets/js/codemirror.xml.js'),
             require.resolve('../assets/js/codemirror.mustache.js'),
             require.resolve('../assets/js/showdown.js'),
-            require.resolve('@mapbox/sphericalmercator')
+            require.resolve('@mapbox/sphericalmercator'),
+            require.resolve('../assets//js/tablesort.min.js')
```

Comment:
Double slash here is not hurting anything but is confusing and should be removed.